client/src/components/TransactionsList.js

```diff
@@ -79,7 +79,7 @@ export default function TransactionsList({ data, fetchTransctions, setEditTransa
               month.transactions.map((row) => (
                 <TableRow key={row._id} sx={{ "&:last-child td, &:last-child th": { border: 0 } }}>
                   <TableCell align="center" component="th" scope="row">
-                    {row.amount}
+                    {`$${row.amount}}
```

Comment:
You did not complete the template literal I merged this PR 
I will do it myself
```
 {`$${row.amount}`}
```

Reply:
oh sorry buddy this is my first pr that's why i am little bit excited and forget to complete template literals.

Reply:
No issue